### DISCUSSION TITLE
Connection refused: no further information

### DISCUSSION THREAD

**Discussion by itsnikhile** [dbeaver/dbeaver#22527] [Q&A]
When im trying to start my workspace to connect with Mysql server (Followed the instructions as well) but it throws this error(in the picture), could someone please help me with the problem 
![2](https://github.com/dbeaver/dbeaver/assets/108937662/3b7a3bec-b04d-4608-b067-d357a1a27ea6)
![1](https://github.com/dbeaver/dbeaver/assets/108937662/84a09501-0288-46c2-b1bd-a588ba7f8cd8)

**Comment by LonwoLonwo** [comment]
Hello @itsnikhile 

What hostname did you use in the hostname field? Sometimes, it is better to use 127.0.0.1 instead of localhost and vice versa.

**Reply by itsnikhile** [reply]
Thank you for the response!
I have tried doing that as well but its still showing the same error :(

![3](https://github.com/dbeaver/dbeaver/assets/108937662/4f708e61-87cb-431d-b223-3cc233031dfa)

**Reply by itsnikhile** [reply]
Do I have to change the driver settings or something ?

**Reply by LonwoLonwo** [reply]
Are you sure that the server is on? Does DBeaver have the right to connect to it?

**Reply by itsnikhile** [reply]
Thanks a lot, actually my professor helped me in figuring it :)

**Reply by LonwoLonwo** [reply]
Could you please provide more details about your solution here? For the following users with the same issue :)

**Reply by uglyhurd** [reply]
good afternoon, how did you solve this problem?

**Reply by XuanThinhTu** [reply]
Could you please provide more details about your solution, I just had the same problem as you

**Reply by Rafay74** [reply]
Can anyone please provide the solution for solving this issue1?

**Reply by Vatsal-Soni-codeware** [reply]
You've to install XAMPP and connect to MySQL and then use DBeaver.

**Reply by bhargav1297** [reply]
You need to install mysql in your machine.

Please follow steps as mentioned here: https://www.w3schools.com/mysql/mysql_install_windows.asp

**Comment by itsnikhile** [comment]
Thanks for helping!

**Reply by uglyhurd** [reply]
good afternoon, how did you solve this problem?

**Reply by bhargav1297** [reply]
It resolved by installing mysql my your machine.

Please follow installation steps as mentioned here:https://www.w3schools.com/mysql/mysql_install_windows.asp

After installation, then start using DBeaver
